app/client/src/components/pages/Community.Tabs.Monitoring.js

```diff
@@ -56,11 +50,62 @@ const containerStyles = css`
   }
 `;
 
+const modifiedDisclaimerStyles = css`
+  ${disclaimerStyles};
+  padding-bottom: 0;
+`;
+
 const modifiedErrorBoxStyles = css`
   ${errorBoxStyles};
   text-align: center;
 `;
 
+const modifiedToggleTableStyles = css`
+  ${toggleTableStyles};
+  tfoot {
+    background-color: #f0f6f9;
+    .totals {
+      border-top: 2px solid #dee2e6;
+      th {
+        border: none;
+        &:first-of-type {
+          display: inline-block;
+          margin-left: 46px;
+        }
+      }
+    }
+    .download-links {
+      border-bottom: 1px solid #dee2e6;
+      th {
+        border-top: none;
+        div {
+          display: inline-block;
+          vertical-align: top;
+          width: 50%;
+          &:first-of-type {
+            text-align: left;
+            a {
+              margin-right: 5px;
+            }
+          }
+          &:last-of-type {
+            text-align: right;
+            .download-text {
+              margin-bottom: 5px;
+            }
+            a {
+              margin-left: 5px;
+            }
+          }
+        }
+        span {
+          display: inline-block;
+        }
+      }
+    }
+  }
+`;
+
```

Comment:
These styles (and the corresponding markup) can be simplified – I'll push up an update in a min doing that.